swagger/v1/openapi.json

```diff
@@ -910,157 +871,191 @@
+          }
         }
-      }
-    },
-    "metadata": {
-      "type": "object",
-      "properties": {
-        "filter": {
-          "type": "string",
-          "example": "name='Standard System Security Profile for Fedora'"
+      },
+      "metadata": {
+        "type": "object",
+        "properties": {
+          "filter": {
+            "type": "string",
+            "example": "name='Standard System Security Profile for Fedora'"
+          }
         }
-      }
-    },
-    "host": {
-      "type": "object",
-      "required": [
-        "name",
-        "account_id"
-      ],
-      "properties": {
-        "name": {
-          "name": "string",
-          "example": "cloud.redhat.com"
-        },
-        "account_id": {
-          "type": "string",
-          "example": "649cf080-ccce-4c02-ba60-21d046983c7f"
+      },
+      "host": {
+        "type": "object",
+        "required": [
+          "name",
+          "account_id"
+        ],
+        "properties": {
+          "name": {
+            "type": "string",
+            "example": "cloud.redhat.com"
+          },
+          "account_id": {
+            "type": "string",
+            "example": "649cf080-ccce-4c02-ba60-21d046983c7f"
+          }
         }
-      }
-    },
-    "links": {
-      "type": "object",
-      "properties": {
-        "self": {
-          "type": "string",
-          "example": "https://compliance.insights.openshift.org/profiles"
+      },
+      "links": {
+        "type": "object",
+        "properties": {
+          "self": {
+            "type": "string",
+            "example": "https://compliance.insights.openshift.org/profiles"
+          }
         }
-      }
-    },
-    "benchmark": {
-      "type": "object",
-      "required": [
-        "ref_id",
-        "title",
-        "version"
-      ],
-      "properties": {
-        "ref_id": {
-          "type": "string",
-          "example": "xccdf_org.ssgproject.content_benchmark_RHEL-7"
-        },
-        "title": {
-          "type": "string",
-          "example": "Guide to the Secure Configuration of Red Hat Enterprise Linux 7"
-        },
-        "version": {
-          "type": "string",
-          "example": "0.1.46"
-        },
-        "description": {
-          "type": "string"
+      },
+      "benchmark": {
+        "type": "object",
+        "required": [
+          "ref_id",
+          "title",
+          "version"
+        ],
+        "properties": {
+          "ref_id": {
+            "type": "string",
+            "example": "xccdf_org.ssgproject.content_benchmark_RHEL-7"
+          },
+          "title": {
+            "type": "string",
+            "example": "Guide to the Secure Configuration of Red Hat Enterprise Linux 7"
+          },
+          "version": {
+            "type": "string",
+            "example": "0.1.46"
+          },
+          "description": {
+            "type": "string"
+          }
```

Comment:
Notice how the spacing changed here. If you look at this file without whitespace changes, you can really tell what changed and what didn't.